alma_tests_cacher/cacher.py

```diff
@@ -177,7 +177,12 @@ async def process_repo(
             repo_dir = Path(workdir, repo_dirname)
             if not repo_dir.exists():
                 self.logger.info('Start cloning git repo: %s', repo.url)
-                exit_code, stdout, stderr = clone_git_repo(workdir, repo.url)
+                try:
+                    exit_code, stdout, stderr = clone_git_repo(workdir, repo.url)
+                except Exception as e:
+                    message = f"Cannot clone git repo: {str(e)}"
+                    self.logger.exception(message)
```

Comment:
`logging.exception` gets exception info by itself, you just need to pass a message
```python
self.logger.exception('Cannot clone git repo:')
```